lib/ui/view/workspace/workspace_view/workspace_view.dart

```diff
@@ -33,7 +38,28 @@ class WorkspaceView extends StatelessWidget {
                       child: SingleChildScrollView(
                         physics: ScrollPhysics(),
                         child: model.workspaces.isEmpty
-                            ? Text("No Organization Available!")
+                            ? //Text("No Organization Available!")
+                            Row(
+                                mainAxisAlignment:
+                                    MainAxisAlignment.spaceBetween,
+                                children: [
+                                  WorkSpaceDisplayInfo(
+                                      'assets/logo/zuri_chat_logo.png',
+                                      'Hngi8 x i4G',
+                                      'hngi8.slack.com'),
+                                  IconButton(
+                                      onPressed: () {
+                                        showModalBottomSheet(
```

Comment:
Add the bottom sheet service and the dialog from the model, which should use their respective services